### PR TITLE
Update mcr.cpp

### PR DESCRIPTION
**Pull request by qitianyuu**
this is origin version, com plz

### CODE REVIEW

File: mcr.cpp [hunk 1]
```diff
@@ -1,104 +1,53 @@
...
+			if (turn == false)
+			   game[i][j] = 'X';
+			else 
+			   game[i][j] = 'O';
+			if (isWin(game)){
+				std::cout << "Win!" << std::endl;
+				break; // need to terminate the problem
+			}
+		}
+	if (i==3) // all celles with i=0 to 2 have been inputted above but no winner yet
```
Comment (chenjunyi1999) [inline mcr.cpp]: this function is wrong!

File: mcr.cpp [hunk 2]
```diff
@@ -1,104 +1,53 @@
...
+	char player1 = 'X';
+	char player2 = 'O';
+	bool turn = true; // false for player 1's turn, true for player 2's turn. Player 1 first.
+	std::cout << "X = Player 1" << std::endl << "O = Player 2" << std::endl;
+       for (int n=0; n<9; n++){
+			turn = !turn;  // use the not-operator to change true to false or false to true.
+		    if (turn == false)
+				std::cout << "Player 1: ";
+			else
+				std::cout << "Player 2: ";
```
Comment (CHENSX97) [inline mcr.cpp]: I think it is necessary to verify the input nums.

File: mcr.cpp (hunk 1 above)
Comment (qitianyuu) [reply mcr.cpp]: ok i fix it now!

File: mcr.cpp (hunk 2 above)
Comment (qitianyuu) [reply mcr.cpp]: ok i fix it

**Pull request merged**